### PR TITLE
Add learning path to onboarding docs

### PR DESCRIPTION
**Pull request by catherineluse**

### CODE REVIEW

File: docs/developer/getting-started/concepts.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+# Official Courses
+
+SUSE provides official training on Rancher and K3s. The courses are on the [SUSE community website.](https://community.suse.com/all-courses) However, because Rancher is built on top of Kubernetes and Kubernetes manages containers, it is ideal to have at least some familiarity with containers and Kubernetes before diving into Rancher.
+
+# Learning path
+
+## 1. Terminology
+
+It may be helpful to start with the [onboarding glossary for new hires](https://confluence.suse.com/display/EN/Onboarding+and+New+Hire+Glossary) and/or the [official Kubernetes glossary.](https://kubernetes.io/docs/reference/glossary/?fundamental=true)
```
Comment (nwmac) [inline docs/developer/getting-started/concepts.md:17]: The confluence document is not available externally - I'd remove it for now.

File: docs/developer/getting-started/terminology.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Terminology
 
-This is part of the developer [getting started guide](./README.md). 
+This is part of the developer [getting started guide](./README.md).
+
+A longer list of terminology for onboarding new hires is in the [Rancher Engineering Confluence site.](https://confluence.suse.com/display/EN/Onboarding+and+New+Hire+Glossary)
```
Comment (nwmac) [inline docs/developer/getting-started/terminology.md:5]: The confluence document is not available externally - I'd remove it for now.

**Review by nwmac — CHANGES_REQUESTED** [review]
LGTM - There are a few links to internal SUSE Confluence - these aren't available externally, so I would remove them for now.

File: docs/developer/getting-started/concepts.md (hunk 1 above)
Comment (catherineluse) [reply docs/developer/getting-started/concepts.md:17]: removed

File: docs/developer/getting-started/development_environment.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ Nuxt Get Started | https://nuxtjs.org/docs/2.x/get-started/installation
 Nuxt Structure | https://nuxtjs.org/docs/2.x/directory-structure
 Axios (HTTP Requests) | https://axios.nuxtjs.org/options
 HTTP Proxy middleware | https://github.com/nuxt-community/proxy-module (https://github.com/chimurai/http-proxy-middleware)
+Conceptual docs about Kubernetes, Rancher GitHub repositories and Rancher frameworks | https://confluence.suse.com/pages/viewpage.action?pageId=771949331
```
[the remark this answers is not in the document]
Comment (catherineluse) [reply docs/developer/getting-started/development_environment.md:24]: removed

File: docs/developer/getting-started/terminology.md (hunk 2 above)
Comment (catherineluse) [reply docs/developer/getting-started/terminology.md:5]: removed

**Pull request merged**